### ISSUE TITLE
Option to globally add more request details

### ISSUE THREAD

**Issue by jurgenhaas** [Gizra/logs_http#4]
This is a nice module and it is delivering great value for the sites I'm developing on Pantheon but surely also on other platforms.

Just wondering if it was a good idea to add an option in the settings to generally add additional information like $GLOBALS, $_GET and $_POST into the events that get submitted to Loggly. If you'd support the idea, I'll be happy to come back with a PR.

**Comment by amitaibu** [comment]
@jurgenhaas Thanks :)

Problem is that it's going to be an expensive post request, as it will have a lot of info. So I think it's better to keep it as slim as possible

**Comment by jurgenhaas** [comment]
@amitaibu Yes, that's right. Therefore I thought that a checkbox to turn that on only temporarily for debugging would be a good way of dealing with it.

**Comment by amitaibu** [comment]
Sure, a `debug` checkbox which is false by default sounds good.
